### PR TITLE
Queriable -> Queryable

### LINKED ISSUE #71: Incorrect spelling of Queriable trait name
The correct name would be *Queryable*.

### PR DESCRIPTION
**Pull request by mfpiccolo**
Closes #71

### CODE REVIEW

**Comment by mfpiccolo** [comment]
@sgrif I aliased the attribute but the Queriable trait was public so do we need to alias that as well?

Something like?
```
trait Queriable<ST: NativeSqlType>: Queryable<ST, Row=FromSqlRow<ST>> {
   fn build(row: Self::Row) -> Self;
}
```

**Comment by sgrif** [comment]
Let's just kill the alias, this is a mechanical change for migrating

**Comment by sgrif** [comment]
This needs a `CHANGELOG` entry.

**Comment by mfpiccolo** [comment]
Removed alias and added changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
 
 ### Changed
 
+* Rename both the `#[derive(Queriable)]` attribute and the `Queriable` trait to use
+  the correct spelling `Queryable`.
+
+## [0.4.1] 2016-01-14
+
+### Changed
```
Comment (sgrif) [inline CHANGELOG.md]: I feel like this is wrong...

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,11 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
 
 ### Changed
 
+* Rename both the `#[derive(Queriable)]` attribute and the `Queriable` trait to
+  use the correct spelling `Queryable`.
+
+### Changed
```
Comment (sgrif) [inline CHANGELOG.md]: Still :scissors: XD

Comment (mfpiccolo) [inline CHANGELOG.md]: :scissors: :scissors:

**Comment by sgrif** [comment]
Good to merge. Needs rebase.

**Pull request merged**